CookieMonster/src/main/java/cmonster/browsers/ChromeBrowser.java

```diff
@@ -291,7 +294,7 @@ protected DecryptedCookie decrypt(EncryptedCookie encryptedCookie) {
                 decryptedBytes = cipher.doFinal(ciphertextTag);
             }
             catch (Exception e) {
-                throw new IllegalStateException("Error decrypting", e);
+                decryptedBytes = null;
```

Comment:
This matches the exception-handling behavior for Linux & Mac (the cookie is skipped).